VERSION

```diff
@@ -94,28 +94,28 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=80:0:0
-libmpi_mpifh_so_version=80:0:0
-libmpi_usempi_tkr_so_version=80:0:0
-libmpi_usempi_ignore_tkr_so_version=80:0:0
-libmpi_usempif08_so_version=80:0:0
-libopen_pal_so_version=80:0:0
-libmpi_java_so_version=80:0:0
-liboshmem_so_version=80:0:0
-libompitrace_so_version=80:0:0
+libmpi_so_version=80:0:40
+libmpi_mpifh_so_version=80:0:40
+libmpi_usempi_tkr_so_version=80:0:40
```

Comment:
Jeff was pretty sure that the Fortran interfaces other than mpif.h were not compatible.  I would feel more comfortable not bumping their age until testing is done.

Reply:
Thanks for the review.  @janjust said that he'd be able to do some level of compatibility testing between v4.1.x and this PR.